course/views.py

```diff
@@ -1224,16 +1256,38 @@ def grant_exception_stage_3(pctx, participation_id, flow_id, session_id):
                     kind=flow_rule_kind.grading,
                     rule=new_grading_rule)
                 fre_grading.save()
+                exception_created = True
 
             # }}}
 
-            messages.add_message(pctx.request, messages.SUCCESS,
-                    ugettext(
-                        "Exception granted to '%(participation)s' "
-                        "for '%(flow_id)s'.")
-                    % {
-                        'participation': participation,
-                        'flow_id': flow_id})
+            if exception_created:
```

Comment:
Wouldn't this be better as two independent messages--"grading exc created" and "access exc created"?

Reply:
Yes, that's better. I'll make the change.

Reply:
Done by #572